src/views/Clients/ViewClients/index.jsx

```diff
@@ -31,9 +36,28 @@ import clientsQuery from './clients.graphql';
 }))
 export default class ViewWorker extends PureComponent {
   state = {
-    clientSearch: '',
+    clientSearch: this.props.user ? this.props.user.credentials.clientId : '',
+    clientId: this.props.user ? this.props.user.credentials.clientId : '',
```

Comment:
Rename to `clientId` to `previousClientId` otherwise it can be difficult to differentiate between `state.clientId` and `props.users.credentials.clientId`.